app/components/primer/dropdown/menu.ts

```diff
@@ -0,0 +1 @@
+import '@github/details-menu-element'
```

Comment:
Hrmm I'm still unable to test the behavior in previews.. 🤔

Reply:
here they are looking fine 🤔